lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
OK, we can separate out using a button in dropdown in a future PR.

I think the current workaround of manually adding a button in the markup will suffice for now.  A separate component might be easiest to implement.